docs-src/references/dynamic-configuration.md

```diff
@@ -77,33 +77,34 @@ Not defining constraints on a dynamic configuration key sets the values globally
   ```yaml
   frontend.rps: # Rate limit on the number of requests sent to the Frontend host.
     - constraints: {} # Sets default value that applies to all Namespaces
-      value: 2000 # The defaul value for this key on the history host is 3000.
+      value: 2000 # The default value for this key on the Frontend host is 3000.
   - constraints: {namespace: "namespace1"} # Sets value for "namespace1" Namespace
-      value: "4000"
+      value: 4000
     - constraints: {namespace: "namespace2"}
-      value: "1000"
+      value: 1000
   ```
 
 - For keys that can be customized at a Task Queue level, you can specify Task Queue name and Task type in addition to Namespace.
   To set values at a Task Queue level, use `taskqueueName` (String) with `taskType` (optional; supported values: `Workflow` and `Activity`).
 
-  For example to override the default maximum queries per second made to the Persistence database from the Frontend Service, add the following to your dynamic configuration file.
+  For example if you have Workers creating a large number of Workflow and Activity tasks per second, you can add more partitions to your Task Queues (default is 4) to accomodate the high number of polls.
+  To do this, add the following to your dynamic configuration file.
 
   ```yaml
-  frontend.persistenceMaxQPS:
+  matching.numTaskqueueReadPartitions:
   - constraints: {namespace: "namespace1", taskQueueName: "tq"}  # Applies to the "tq" Task Queue for both Workflows and Activities.
-    value: 3500 # The default value for this key on the Frontend Service is 2400.
+    value: 8 # The default value for this key is 4. Task Queues that need to support high traffic require higher number of partitions. Set these values in accordance to your poller count. 
   - constraints: {namespace: "namespace1", taskQueueName: "other-tq", taskType: "Activity"} # Applies to the "other_tq" Task Queue for Activities specifically.
-    value: 4000 
+    value: 20 
   - constraints: {namespace: "namespace2"}  # Applies to all task queues in "namespace2".
-    value: 1500
+    value: 10
   - constraints: {}  # Applies to all other task queues in "namespace1" and all other Namespaces.
-    value: 3000
+    value: 16
   ```
 
-Note that the values set with most constraints take priority over values that are set with fewer constraints, regardless of the order in which they are set in the dynamic configuration key.
+<!-- Note that the values set with most constraints take priority over values that are set with fewer constraints, regardless of the order in which they are set in the dynamic configuration key. -->
```

Comment:
yeah, maybe it's okay to leave it out. people can ask through support if they get really confused